src/main/java/core/basesyntax/impl/ReportCreator.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.service.CreateReport;
+import java.util.Map;
+
+public class ReportCreator implements CreateReport {
```

Comment:
use noun for class and interface names 
```suggestion
public class ReportCreatorImpl implements ReportCreator {
```